docs/user/library/referencing/oracle.rst

```diff
@@ -91,7 +91,7 @@ The following connection management facilities are provided:
 Example using the popular ``commons-dbcp`` implementation of DataSource::
     
     BasicDataSource source = new BasicDataSource();
-    source.setDriverClassName("oracle.jdbc.driver.OracleDriver");
+    source.setDriverClassName("oracle.jdbc.OracleDriver");
```

Comment:
TODO: We should make a note of this in our updating.rst file

Reply:
The old class is still there in ojdbc8/19.3.x, but was deprecated some years ago.
When upgrading  to ojdbc10 (java 10+) or a higher version eg. 20.??.x it may disappear.